### PR TITLE
Replace static methods #18

### PR DESCRIPTION
**Pull request by odan**
This PR replaces the static Session method with non-static methods.

Affected methods:

* `RKA\Session::regenerate()`
* `RKA\Session::destroy()`

See #18

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /vendor
 /build
 /.phpunit.result.cache
+/.idea/
```
Comment (akrabat) [inline .gitignore]: Really, this should be in your global gitignore, not in every project you ever touch. I'm not opposed to it though.

File: tests/RKATest/SessionTest.php [hunk 2]
```diff
@@ -76,7 +77,9 @@ public function testDestroy()
     {
         @session_start();
         $this->assertEquals(PHP_SESSION_ACTIVE, session_status());
-        @Session::destroy(); // silence headers already sent warning
+        $session = new Session;
+        $session->destroy();
+        // silence headers already sent warning
```
Comment (akrabat) [inline tests/RKATest/SessionTest.php:82]: I suspect we should move this comment to above `@session_start()` or just remove it.

**Review by akrabat — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (odan) [reply .gitignore]: I removed it.

File: tests/RKATest/SessionTest.php (hunk 2 above)
Comment (odan) [reply tests/RKATest/SessionTest.php:82]: This was fixed later, but should be part of the PR.

https://github.com/akrabat/rka-slim-session-middleware/pull/20/files#diff-9f266881fe264e73745b1641b8af3ea8c6df044d2ebf3679c8c941a1f47d35d0R80

**Comment by akrabat** [comment]
Why did you close this?

**Comment by odan** [comment]
I think I've made something not correct in my PR, so I guess that's why I closed it.
Should we open it again?

**Pull request closed without merging**